src/pages/home/report/ContextMenu/genericReportActionContextMenuPropTypes.js

```diff
@@ -25,7 +28,6 @@ const propTypes = {
 const defaultProps = {
     isMini: false,
     isVisible: false,
-    selection: '',
```

Comment:
Can't remove this. selection is not required prop.

Reply:
You marked this resolved but I don't see any change. Did you check the console errors?

Reply:
Because above, you said to treat this as null. I have not encountered errors.

Reply:
Do you mean linter errors? I don't get any, but I think the linter warns about this scenario elsewhere. I can just add `selection: null`.

Reply:
Added `selection: null` to default props. Let me know if you meant something else.

Reply:
To elaborate, actions on the videos and other actions I did do not produce runtime errors. Linter errors that occur in other components (not in this one for some reason) in similar cases are resolved just by adding the property to `defaultProps`, which I have done in the last commit.